### DISCUSSION TITLE
Globally stop U-Turns

### DISCUSSION THREAD

**Discussion by Coding-Andy** [krzychu124/Traffic#4] [Ideas]
I would like to see an option to outright stop or at the least significantly reduce u-turns.

Random u-turns cause unnecessary traffic issues in cities. This isn't just at intersection, but a random points on roads.

**Comment by Programmer-Timmy** [comment]
> I would like to see an option to outright stop or at the least significantly reduce u-turns.
> 
> Random u-turns cause unnecessary traffic issues in cities. This isn't just at intersection, but a random points on roads.

Hi, you can try and use this mod for now. It can change to cost for making u turns and more! Then you make it less attractive to do that movement.

https://mods.paradoxplaza.com/mods/86462/Windows

**Reply by KuristZero** [reply]
Hey Timmy, also would like to see this in the settings menu if possible?
Like how you have the button to remove U-Turns and Unsafe in the lane connector tool, it would be amazing to have a global button in settings which applies it to every intersection.

I understand this might be a bit tricky, as I suspect this would require the mod to be able to find every intersection in the game area and then apply the rule to them one by one, but if it's possible it would be an amazing addition to the tool - especially with IceFlake now looking to improve the game :)

No stress if it's too much effort, as you mentioned the one above can be used - but it would be an amazing feature!
Keep up the amazing work!

**Reply by Programmer-Timmy** [reply]
Hey @KuristZero,

I didn’t contribute to this mod myself, and I don’t really know the codebase. I’ve also kind of stopped modding for Cities: Skylines 2 because I don’t feel it’s worth my time with the current state of the game. I hope IceFlake improves the game enough for more modders to become active again and make the time investment worthwhile.

You could ask the original code owner of this repo, @KuristZero, what they think of the idea, or try contributing yourself.

I hope this helps. Just let me know if you have any more questions!
